packages/instrumentation-sequelize/test/sequelize.spec.ts

```diff
@@ -292,6 +292,71 @@ describe('instrumentation-sequelize', () => {
     });
 
     describe('config', () => {
+        describe('queryHook', () => {
+            it('able to collect query', async () => {
+                instrumentation.disable();
+                const instance = new sequelize.Sequelize(`postgres://john@$localhost:1111/my-name`, { logging: false });
+                instance.define('User', { firstName: { type: sequelize.DataTypes.STRING } });
+
+                const response = { john: 'doe' };
+                sequelize.Sequelize.prototype.query = () => {
+                    return new Promise((resolve) => resolve(response));
+                };
+                instrumentation.setConfig({
+                    queryHook: (span: Span, {sql, option}: {sql: any, option: any}) => {
+                        span.setAttribute('test-sql', 'any');
+                        span.setAttribute('test-option', 'any');
```

Comment:
I was unsure how best to test `sql` and `option` since their type is unknown to me. (Alas, I couldn't run the tests on my laptop.)

Reply:
I can help you run the tests locally, and then you can examine the value in those two parameters.
How are you running the tests, and what errors are you seeing?

Reply:
I ran `npm run test` in the `packages/instrumentation-sequelize` directory. It said "ENOENT: no such file or directory" looking for `jaeger-client`. I'm OK with letting CI run the tests 🙂 .

Reply:
You need to first `npm run build` in the root of the repo to build the test-utils package which is used for running the tests

Reply:
Yep, that worked, I was able to run the tests locally.